qutebrowser/utils/qtutils.py

```diff
@@ -132,14 +132,16 @@ def get_args(namespace):
     for argname, val in vars(namespace).items():
         if not argname.startswith('qt_'):
             continue
```

Comment:
Before, there was this for-loop here which handled any argument starting with `--qt-` (which the `argparse` library transforms to `qt_`). Now you only need to handle `--qt-arg` (`namespace.qt_arg`) and `--qt-flag` (`namespace.qt_flag`) explicitly, so you should be able to get rid of almost the whole old logic.